### ISSUE TITLE
Add to xlr-xldeploy-plugin?

### ISSUE THREAD

**Issue by tombatchelor** [xebialabs-community/xlr-xldeploycli-plugin#1]
This looks awesome, but wouldn't this be better as a task type in the existing xlr-xldeploy-plugin? Seems that as that has the XL Deploy server CI already it could help avoid some duplication.

**Comment by zvercodebender** [comment]
Good point.  I'll remove this repo and put my code in xlr-xldeploy-plugin
